hedyorg/hedy
# [CHORE] Make Update JavaScript action run in PR context

The event `pull_request_target` (as opposed to `pull_request`) makes it run in the context of `main`, instead of the context of the PR itself.

This caused it to use files from `main` instead of the PR, incorrectly representing  its success or failure.

**How to test**

On this PR:

* We should be able to change a TS file and see the JS be regenerated and committed.

After merging:

* The same should happen for fork PRs.

In case this action is broken for fork PRs, we should just add something like:

```
if: github.event.push
```

To the commit action, and forego committing on PRs altogether (just leave committing on main), but DO run the generator, so that we can see it if it fails.
